applications/parachain-staking.md

```diff
@@ -0,0 +1,80 @@
+
+### Background
+
+In the Polkadot consensus model, parachains have different requirements and constraints than the relay chain. While `frame/pallet-staking` offers many features necessary for relay chain consensus and shared security, it is overkill for parachains, which operate with more limited execution resources.
+
+Instead of running an on-chain election, [`parachain-staking`](https://github.com/PureStake/moonbeam/tree/master/pallets/parachain-staking/src) implements direct delegation with a bounded number of nominations per `AccountId` (maximum of `Config::MaxCollatorsPerNominator` per account). In this paradigm, token holders (nominators) express exactly which collator candidates they would like to support and with what quantity of stake.
+
+There is a new round every `<Round<T>>::get().length = 600` blocks. At the start of each round, the top `<TotalSelected<T>>::get() = 16` collator candidates (in terms of total backing stake) are chosen to be in the active collator set. At every block, a subset of this active collator set is chosen pseudorandomly by the `author-filter` pallet as valid block authors. Only valid block authors can produce blocks.
+
+Block authors are logged via the `set_author_inherent` method in the `author-inherent` pallet. The logic in this runtime method reports each block author to `parachain-staking`, which distributes inflation rewards after a 2 round delay.
+
+## Team :busts_in_silhouette:
+
+* **Members:** Derek Yoo, Alan Sapede, Antoine Estienne, Joshy Orndorff, Amar Singh
+* **LinkedIn Profiles**: https://www.linkedin.com/in/derek-yoo-8a050/, https://www.linkedin.com/in/alansapede/,
+* **Code Repos:** https://github.com/PureStake/moonbeam
+* **Website:** https://www.purestake.com/, https://moonbeam.network/
+* **Legal Structure:** PureStake Ltd, Floor 4, Willow House, Cricket Square, Grand Cayman, KY1-9010, Cayman Islands
+* **Team's Experience:** 
+
+The PureStake team has extensive experience developing and delivering complex web2 software systems. In the last year, PureStake has built a number of web3 infrastructure projects including Algorand API Services (https://developer.purestake.io/) and the Goalseeker block explorer (https://goalseeker.purestake.io/), among others.
+
+Derek co-founded Fuze in 2006 and as CTO was responsible for engineering, technical operations, product management, and design as the company grew over time into a 700 person cloud business. Alan was responsible for and hands on developed large parts of the Fuze cloud backend. Before joining Fuze, Alan was on the Google Streetview Team and also worked at NVidia developing drivers. Joshy and Amar worked at Parity on the Substrate Recipes.
+
+* TODO: Antoinne experience
```

Comment:
Eth derivation in Polkadot-js common and other contributions to Polkadot-js family plus contributions to polkadot-launch.

Worked at consensys before that, not sure one what @joelamouche

Reply:
Thanks @JoshOrndorff 
"Dapp Developer since 2016, Antoine has participated in 3 projects inside ConsenSys (VariabL, Localties and Fathom) and helped HyperNetwork build their token system. Now, he's working for Moonbeam and has contributed to the polkadot-js suite (apps, common, api...)."